pkg/nar/util_test.go

```diff
@@ -8,43 +8,54 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestLexicographicallyOrdered(t *testing.T) {
-	testCases := []struct {
-		path1    string
-		path2    string
-		expected bool
-	}{
-		{
-			path1:    "/foo",
-			path2:    "/foo",
-			expected: true,
-		},
-		{
-			path1:    "/fooa",
-			path2:    "/foob",
-			expected: true,
-		},
-		{
-			path1:    "/foob",
-			path2:    "/fooa",
-			expected: false,
-		},
-		{
-			path1:    "/cmd/structlayout/main.go",
-			path2:    "/cmd/structlayout-optimize",
-			expected: true,
-		},
-		{
-			path1:    "/cmd/structlayout-optimize",
-			path2:    "/cmd/structlayout-ao/main.go",
-			expected: false,
-		},
-	}
+// nolint:gochecknoglobals
```

Comment:
are there cases where this lint makes sense? It might be easier to set this globally otherwise.

Reply:
I think as a rule it makes sense to avoid global variables and we shouldn't introduce new ones without thinking twice, a linter rule is a good barrier I think.